contracts/CompoundGovernor.sol

```diff
@@ -116,6 +116,37 @@ contract CompoundGovernor is
         _setProposalGuardian(_proposalGuardian);
     }
 
+    /// @notice Cancels an active proposal.
+    /// @notice This function can be called by the proposer, the proposal guardian, or anyone if the proposer's voting
+    /// power has dropped below the proposal threshold. For whitelisted proposers, only the whitelist guardian can
+    /// cancel if the proposer is below the threshold.
+    /// @param targets An array of addresses that will be called if the proposal is executed.
+    /// @param values An array of ETH values to be sent to each address when the proposal is executed.
+    /// @param calldatas An array of calldata to be sent to each address when the proposal is executed.
+    /// @param descriptionHash The hash of the proposal's description string.
+    /// @return uint256 The ID of the canceled proposal.
+    function cancel(
```

Comment:
note: this will also have to be reproduced for the `cancel(uint proposalId)` method in #3.

Reply:
created an issue so we don't forget https://github.com/ScopeLift/compound-governance-upgrade/issues/37